relay-sampling/src/lib.rs

```diff
@@ -665,13 +661,9 @@ impl SamplingConfig {
             RuleType::Error
         };
 
-        for rule in &self.rules {
-            if rule.ty == ty && rule.condition.matches(event, ip_addr) {
-                return Some(rule);
-            }
-        }
-
-        None
+        self.rules
+            .iter()
+            .find(|&rule| rule.ty == ty && rule.condition.matches(event, ip_addr))
```

Comment:
Same above:

```suggestion
            .find(|rule| rule.ty == ty && rule.condition.matches(event, ip_addr))
```

Reply:
done in [3cd51a9](https://github.com/getsentry/relay/pull/1493/commits/3cd51a9665eb1a7c8fdc84920cc6d3afaddbed07)